tools/monitor.js

```diff
@@ -0,0 +1,87 @@
+import Arweave from 'arweave';
+import { connect, createDataItemSigner } from '@permaweb/aoconnect';
+import {
+  ANTRegistry,
+  AOProcess,
+  ANT_REGISTRY_ID,
+  ARIO,
+  ARIO_MAINNET_PROCESS_ID,
+} from '@ar.io/sdk';
+import { pLimit } from 'plimit-lit';
+
+const arweave = Arweave.init({
+  host: 'arweave.net',
+  port: 443,
+  protocol: 'https',
+});
+
+async function main() {
+  const jwk = await arweave.wallets.generate();
+
+  const ao = connect({
+    CU_URL: 'https://cu.ardrive.io',
```

Comment:
i'd prefer this spun up a local CU

Reply:
Im not sure which is more finite, cu resources or our gh action minutes, wdyt?